### ISSUE TITLE
ES6: Destructuring from Nested Objects solution is incorrect

### ISSUE THREAD

**Issue by joshalling** [freeCodeCamp/guide#8416]
The given solution is incorrect in the hint for the ES6: Use Destructuring Assignment to Assign Variables from Nested Objects Challenge. It currently uses the `LOCAL_FORECAST` global which will cause your code to return 84.6 regardless of what is passed to the function.

**Incorrect:**
```javascript
const { tomorrow: { max: maxOfTomorrow } } = LOCAL_FORECAST;
```

**Correct:**
```javascript
const { tomorrow: { max: maxOfTomorrow } } = forecast;
```
This would be a good contribution for any beginners or first-timers.

---

> ✅️ By submitting this issue, I have verified the following

- [x] [Checked](https://github.com/freeCodeCamp/guides/issues?q=is%3Aissue+is%3Aclosed) to see if the issue has already been discussed before. 🤔️
- [x] *If* proposing new content to be added, made sure enough details were provided. 🔍️

**Comment by GlynL** [comment]
I'd like to take this issue on if that's okay.

**Comment by Asjas** [comment]
@GlynL :+1:

**Comment by JimmayVV** [comment]
Just going through the open Issues on this project, it appears this issue should be closed as per #8423 - correct? 😄

**Comment by joshalling** [comment]
That's correct. Thanks for the heads up @JimmayVV!
